src/sql/workbench/browser/modal/optionsDialog.ts

```diff
@@ -204,16 +203,10 @@ export class OptionsDialog extends Modal {
 			let bodyContainer = $('table.optionsDialog-table');
 			this.fillInOptions(bodyContainer, serviceOptions);
 			append(this._optionGroupsContainer!, bodyContainer);
-
-			if (!firstOption) {
```

Comment:
This not needed or something?

Reply:
oh, yes, forgot to mention, the move focus to the first focusable element is now available in the base modal class, no longer needed here.